src/cloudai/cli/handlers.py

```
@@ -142,14 +142,14 @@ def handle_dse_job(runner: Runner, args: argparse.Namespace) -> int:
             test_run = copy.deepcopy(tr)
 
             agent_type = test_run.test.agent
-            agent_class = registry.agents_map.get(agent_type)
-            if agent_class is None:
+            if not registry.has_agent(agent_type):
                 logging.error(
                     f"No agent available for type: {agent_type}. Please make sure {agent_type} "
-                    f"is a valid agent type. Available agents: {registry.agents_map.keys()}"
+                    f"is a valid agent type. Available agents: {registry.agent_names()}"
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)